packages/host/app/services/operator-mode-state-service.ts

```diff
@@ -250,6 +252,19 @@ export default class OperatorModeStateService extends Service {
     }
   }
 
+  replaceCodePath(codePath: URL | null) {
```

Comment:
Can we name this to `replaceCodePathAfterRedirection` for extra clarity?

Reply:
I did talk to hassan about this -- to rename this function to hint that its for a redirect. And I think we discuss, this essentially calls `replaceWith` which doesn't necessarily mean redirect; it cud mean just forget about the last `window.history` state.